### ISSUE TITLE
App still is crashing now after upgrading to 1.8.11 #2031 and only happens with my original restore phrase 

### ISSUE THREAD

**Issue by wksantiago** [get10101/10101#2075]
This is an update to closed issue #2031 

App still is shutting down now after upgrading to v1.8.11 #2031 and only happens with my original restore phrase.

I created a new phrase and the app start OK no problem.  I clear storage and cache and restore from original phrase and then goes form:

Welcome 10101 to
...
Starting 10101

Then it just shuts down.

**Comment by holzeis** [comment]
Thanks for opening another issue. 

Does the state that you are trying to restore includes a channel?

**Comment by wksantiago** [comment]
The original a few weeks ago yes I had a channel for the stable token.

and I am running the apk form github

**Comment by bonomat** [comment]
Thanks for letting us know. 
In case you know your way around in a terminal, could you connect your phone and run`adb logcat` (https://developer.android.com/tools/adb) and send us the logs. This would be tremendously helpful for debugging your case.

**Comment by bonomat** [comment]
Thank you for this. 
This is the critical part: 

```
02-22 16:05:39.380 29178 29221 I flutter : [ERROR] TIME: 2024-02-22T20:05:39.380657Z Launching the app failed FfiException(RESULT_ERROR, Failed to start the backend
02-22 16:05:39.380 29178 29221 I flutter :
02-22 16:05:39.380 29178 29221 I flutter : Caused by:
02-22 16:05:39.380 29178 29221 I flutter :     0: Failed to get wallet balances
02-22 16:05:39.380 29178 29221 I flutter :     1: Could not find contract associated with channel to compute usable balance, null)
02-22 16:05:39.381 29178 29221 I flutter : [DEBUG] TIME: 2024-02-22T20:05:39.381026Z r: Got all spendable outputs outputs: []
02-22 16:05:39.389 29178 29221 I flutter : [DEBUG] TIME: 2024-02-22T20:05:39.389418Z r: Got all transactions transactions: []
02-22 16:05:39.389 29178 29221 I flutter : [DEBUG] TIME: 2024-02-22T20:05:39.389750Z r: Syncing 0 shadow transactions
```

**Comment by bonomat** [comment]
We will look into it, I've prioritized it for this week.

**Comment by holzeis** [comment]
The issue resulted out of a missing contract backup which fails wallet sync at startup. This let the app not start properly and crashed the app.

#2090 adds a more graceful handling to that so that the app will not crash at startup. However, the off-chain balance will not be able to get calculated because of the missing contract rendering the balance after the fix to "n/a".

**Comment by wksantiago** [comment]
Will the contract eventually close?

-------- Original Message --------

**Comment by holzeis** [comment]
> Will the contract eventually close?

The contract should already be in closing on-chain, due the force closure. Fixing that issue will allow your app to start again so that it can monitor the buffer transaction so that once confirmed it can sweep the funds.

**Comment by wksantiago** [comment]
After successfully upgrading to v1.9.0 the issue is still happening.

Should I attach another log or maybe even install the app on a Google phone since this is a CalyxOS pixel?

**Comment by holzeis** [comment]
> After successfully upgrading to v1.9.0 the issue is still happening.

Uff don't update your app before we release the coordinator. This is a bit of a shortcoming of our apk release. We will have to think of a way to communicate that properly.

> Should I attach another log or maybe even install the app on a Google phone since this is a CalyxOS pixel?

The fix was not part of 1.9.0 yet.   The patch is still in review. Once approved it will be part of 1.9.1.

**Comment by holzeis** [comment]
@wksantiago I am reopening this ticket, so we don't forget to get your confirmation that this issue has been resolved for you once the new release is out.

**Comment by wksantiago** [comment]
Yes it opens to a window indicating “Collaborative Channel Close! Failure Your channel has been closed collaboratively!”

Under the Trade tab it indicates Closing...

Should I transfer the on-chain sats to another wallet?

**Comment by holzeis** [comment]
The position is definitely closed and just not updated in your app. You can go to Settings > Emergency Kit and cleanup your data.

- Delete position
- Cleanup filling orders

If you don't see your funds in your wallet, it might be still in the split or buffer transaction wait for confirmations. If you share your logs with us, we can further investigate whats the status of your app.

You share your logs at Settings > Share Logs.

However, chances are that through the restore, we might have lost data, which would have allowed your app to redeem the funds. But lets hope for the best.

**Comment by wksantiago** [comment]
Here is the log file Logs from 2024-03-12_114821 
[2024-03-12_114821.log](https://github.com/get10101/10101/files/14575205/2024-03-12_114821.log)

**Comment by holzeis** [comment]
> After making the recommendations above all went well and no positions, but when opening the app after closing I get the same prompt “Collaborative Channel Close! Failure Your channel has been closed collaboratively!”
> 
> Here is the log file Logs from 2024-03-12_114821 [2024-03-12_114821.log](https://github.com/get10101/10101/files/14575205/2024-03-12_114821.log)

The prompt should be gone now. We are keep investigating the state of your app, and will let you know once we know more.

**Comment by holzeis** [comment]
Hi @wksantiago !

This is the state of your [channel](https://mempool.space/tx/8e9cf38acbef789d0f4abfaafbbeaa3721bc15393af16ff55fb116d0aa096c68) at the moment.

The [split tx](https://mempool.space/tx/a8ea77c3278b1e9cce099b2e7136a4175e3121464aa1075db9e58fd1e242840a) got broadcasted and confirmed. It split up your lightning channel into a dlc and a lightning channel.


### Lightning Channel

The [glue tx](https://mempool.space/tx/38c620b56661e68f33dde83c8f30b50fff82a7a37e366241a93ab623bfedadc5) with an input of 490,075 sats also got broadcasted and confirmed.

The [commitment tx ](https://mempool.space/tx/45585e03a011a3bffd678563374dd198a319f03303b08802c758db61106ecf59) with an input of 488,392 sats got broadcasted and confirmed.

Your [claim tx](https://mempool.space/tx/3f256bb60aaa10d59884e8aea7bf4f2ccdbdcbf3daee6c2e20af1fd9e724a2fc) got broadcasted and confirmed. I presume now that you app can startup again you have been able to claim your pending funds. You should see these funds already in your wallet.


### DLC Channel

The [buffer tx](https://mempool.space/tx/0efafd1d060552113dc7a14dfd92ccb52965db887bf13004d0590bfe71f8af9f) with an input of 183,188 sats has been broadcasted and confirmed.

The [CET](https://mempool.space/tx/d2b38531ec7c3e89e6c2485cac6b294d6e259e4f689b2fb8e86b068884efef38) has also already been broadcasted but is still pending. The fee is pretty low, but it should be possible to CPFP that. You should be able to do that by sending the max amount of your wallet to yourself. This transaction will then pay as well for the CET so make sure to choose a higher fee rate to be able to pay for both transactions.


![Image](https://github.com/get10101/10101/assets/382048/c73f0835-b98f-424e-ae27-12639f012275) [^1]

[^1]: https://10101.finance/blog/dlc-to-lightning-part-3

**Comment by wksantiago** [comment]
Great stuff thanks I will keep an eye on the translation on the main chain

**Comment by holzeis** [comment]
I am closing this ticket. Feel free to open another ticket if you need anymore support.
